packages/test/test-end-to-end-tests/src/test/gc/gcTestConfigs.ts

```diff
@@ -0,0 +1,30 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ConfigTypes, IConfigProviderBase } from "@fluidframework/telemetry-utils";
+import { ITestContainerConfig } from "@fluidframework/test-utils";
+
+export const mockConfigProvider = ((settings: Record<string, ConfigTypes>): IConfigProviderBase => {
```

Comment:
```suggestion
export const mockConfigProvider = ((settings: Record<string, ConfigTypes> = {}): IConfigProviderBase => {
```
Seems most places just pass an empty object

Reply:
Change it.